src/app/server/RendezvousServer.h

```diff
@@ -31,11 +32,12 @@ class RendezvousServer : public SessionEstablishmentDelegate
     CHIP_ERROR WaitForPairing(const RendezvousParameters & params, Messaging::ExchangeManager * exchangeManager,
                               TransportMgrBase * transportMgr, SecureSessionMgr * sessionMgr, Transport::AdminPairingInfo * admin);
 
-    CHIP_ERROR Init(AppDelegate * delegate, PersistentStorageDelegate * storage)
+    CHIP_ERROR Init(AppDelegate * delegate, PersistentStorageDelegate * storage, SessionIDAllocator * idAllocator)
     {
         VerifyOrReturnError(storage != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
-        mDelegate = delegate;
-        mStorage  = storage;
+        mDelegate    = delegate;
+        mStorage     = storage;
+        mIDAllocator = idAllocator;
```

Comment:
Should these also be validated as INVALID_ARGUMENT if null? why are we checking only storage?